apps/src/code-studio/projectRedux.ts

```diff
@@ -0,0 +1,115 @@
+import {PayloadAction, createSlice} from '@reduxjs/toolkit';
+
+interface ProjectReduxState {
+  showProjectUpdatedAt: boolean;
+  projectUpdatedStatus: string;
```

Comment:
Could this be `keyof typeof projectUpdatedStatuses` to ensure it's one of those? Might need to add `as const` to the `projectUpdatedStatuses` for that to work. 
Similarly for workspaceAlertTypes?

Reply:
thanks! I didn't know about `as const` so I didn't think this was possible. Fixing!